docs/topics/api/overview.rst

```diff
@@ -98,6 +98,7 @@ The following properties will be available in paginated responses:
 * *next*: the URL for the next page in the pagination.
 * *previous*: the URL for the previous page in the pagination.
 * *page_size*: The number of items per page in the pagination.
+* *page_count*: The number of pages available in the pagination.
```

Comment:
Maybe mention that it may be lower than `count / page_size` because of reasons.

Reply:
Good point, especially since we don't lie about `count`. Fixed in 4e43dcccba